src/Employer/Employer.Web/Program.cs

```diff
@@ -36,6 +36,7 @@ public static void Main(string[] args)
                         if (hostingEnvironment.IsDevelopment())
                         {
                             config.AddUserSecrets<Startup>();
+                            config.AddEnvironmentVariables();

```

Comment:
I think this is redundant (I also think AddUserSecrets is redundant also) as it sounds like it's setup by the fact we use .CreateDefaultBuilder()